### ISSUE TITLE
Bug: "Icon Menu" does not show settings options

### ISSUE THREAD

**Issue by Tronic69** [mrdokenny/Cookie-AutoDelete#358]
After click on CAD icon button for settings is not visible (see screenshot below). 

[Link to screenshot](https://i.imgur.com/p11NsEG.png)

- OS/version: Windows 7 and Linux Mint 18.3 
- Browser/version: Vivaldi 1.15.1099 (Win + Linux) and latest Chrome on Windows 7
- Cookie AutoDelete version: 2.1.2

**Comment by ke-d** [comment]
Looks like it's a problem with the long strings in that locale.

**Comment by Nisc3d** [comment]
I have the same problem. An option to change the language would be nice.

**Comment by kennethtran93** [comment]
I've added the option to change text size in an upcoming version.  This may mitigate the long strings.  Overflow CSS is also set to auto so scrollbars should be visible.

However...
- Chrome doesn't set the popup width correctly as it most likely doesn't take into consideration how react would render the popup.  For this I've set a min-width CSS property based on the English string lengths.
- Changing language will have to be done on the browser end as we are calling `browser.i18n.getMessage` to get the localized versions with no way to set preferred locales (correct me if i'm wrong).  Check your browser on how to change locales. Note that this may also effect the websites if they load based on preferred locales.

In the future...If you only list Vivaldi browser, I won't be looking into it unless you have tested the same issue through Google Chrome/Mozilla Firefox.
